spec/features/staff/users_can_view_an_activity_as_xml_spec.rb

```diff
@@ -7,6 +7,30 @@
     context "when the user belongs to BEIS" do
       before { authenticate!(user: user) }
 
+      context "when the activity has a previous activity identifier" do
+        let(:activity) {
+          create(:fund_activity,
+            organisation: organisation,
+            identifier: "IND-ENT-IFIER",
```

Comment:
For ingested records these fields are the same thing. What was the intent behind how we could populate the `identifier` field in this case?